### DISCUSSION TITLE
EHBASIC

### DISCUSSION THREAD

**Discussion by n9jcv** [picocomputer/community#69] [Q&A] [answered]
So I am not sure I understand everything here.  I want to get EHBASIC running on my new rp6502.  

Is EHBASIC available in compiled/ROM form somewhere that I can just dowload and install?

Or do I have to install all the tools and build myself?
Thanks!

**Comment by vruumllc** [comment, marked as answer]
I uploaded a basic.zip file to the first builds thread. It’s old, but might still work.

**Reply by n9jcv** [reply]
@vruumllc  Thank you very much it works great!!
![1704045424713985394515760063042](https://github.com/picocomputer/community/assets/6472203/c0f91028-819b-405b-b96a-99e9a4ec4691)


 This helps.  I have to crawl, then walk and maybe run.  At some point I need to get the dev tools on windows and start learning that stuff, thanks again!

I am just going to document the steps here for some one else that may be in my same place.

First download the basic zip file, from here
Thanks to @vruumllc 
https://github.com/orgs/picocomputer/discussions/20#discussioncomment-7741295


Then unzip that.  It creates a basic folder.  Put that on a flash drive.  Then put the flash drive on the rp6502.  

cd basic
install basic

Then anytime you want to run basic, just type basic
Enter program and run 
Have Fun!

**Reply by netzerohero** [reply]
Good that you captured steps to load BASIC to ROM.  
I linked your steps to the [picocomputer community wiki.](https://github.com/picocomputer/community/wiki)

**Reply by n9jcv** [reply]
Any small contribution I can make I am more than willing, others are doing the hard work!
@netzerohero

**Comment by netzerohero** [comment]
Welcome to the community!  Congratulations on getting [your board up](https://github.com/orgs/picocomputer/discussions/68#discussion-6014233). 

Attached is a zip containing an rp6502 rom-file along with a couple of demo files.
[basic_20231231.zip](https://github.com/picocomputer/community/files/13801817/basic_20231231.zip)

Info on contents (which is a fork ahead of the rp6502's EhBASIC) can be found [here](https://github.com/picocomputer/ehbasic/issues/2#issuecomment-1868562547) and [here](https://github.com/netzerohero/picocomputer-ehbasic).

Hope this helps!

**Reply by n9jcv** [reply]
Thank you @netzerohero I will have a look at these!

**Comment by rumbledethumps** [comment]
netzerohero is releasing wip builds here:
https://github.com/netzerohero/picocomputer-ehbasic/releases
